mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairCards.tsx

```diff
@@ -296,7 +297,20 @@ interface AvailablePoolProps {
   setFavouritePoolpair: (id: string) => void
 }
 
+type ActionType = 'SET_FAVOURITE' | 'UNSET_FAVOURITE'
```

Comment:
export here then we can import from poolpair detail screen
```suggestion
export type ActionType = 'SET_FAVOURITE' | 'UNSET_FAVOURITE'
```